go-controller/pkg/metrics/master.go

```diff
@@ -56,7 +56,7 @@ var metricPodCreationLatency = prometheus.NewHistogram(prometheus.HistogramOpts{
 	Namespace: MetricOvnkubeNamespace,
 	Subsystem: MetricOvnkubeSubsystemMaster,
 	Name:      "pod_creation_latency_seconds",
-	Help:      "The latency between pod creation and setting the OVN annotations",
+	Help:      "The duration between a pod being scheduled and configuring its logical switch port",
```

Comment:
"configuring" got me a bit confused, since it sounds like a process, I couldn't figure out which point in time exactly it is. 
maybe something around "and ovnkube-master finishing its logical switch port configuration" will be a bit more obvious? (I know it has a context of ovnkube-master, but it took me some time to realize it, so maybe it makes sense to leave it in the description?)

Reply:
Hope its better now :) Didnt stick exactly to what you suggested but I _think_ mine is better.